config/default/kustomization.yaml

```diff
@@ -23,14 +23,10 @@ resources:
 - ../components/internalcert
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 #- ../prometheus
+# [METRICS] Expose the controller manager metrics service.
+- manager_metrics_service.yaml
 
 patchesStrategicMerge:
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
-- manager_auth_proxy_patch.yaml
```

Comment:
@googs1025 this is probably causing #836 

We don't patch our deployment with the metrics ports.